### PR TITLE
use yarn to install, with the lockfile

### PR DESCRIPTION
**Pull request by brunnre8**

### CODE REVIEW

**Comment by brunnre8** [comment]
Switch to yarn like all the others.
Also stops messing with the file permissions inside of /etc/thelounge in order to not expose vapid.json and similar sensitive files.
TL complains to the admins if something is fishy and once they fix it we should be good.

I plan on fixing the "wrong config owner" message in the near future but for the meantime it's better than what we have now.

File: PKGBUILD [hunk 1]
```diff
@@ -1,46 +1,74 @@
 # Maintainer: Reto Brunner <brunnre8@gmail.com>
 # Maintainer: Maxime Poulin <maxpoulin64@gmail.com>
 pkgname=thelounge
-pkgver=4.3.0
-pkgrel=2
+pkgver=4.3.1
+pkgsuffix=-rc.1
+pkgrel=4
```
Comment (maxpoulin64) [inline PKGBUILD:6]: You can reset pkgrel back to 1 whenever pkgver increases, pkgrel is for re-releases of the same package with packaging changes.

**Review by maxpoulin64 — APPROVED** [review]
Makes sense to me, I didn't even realize we were still using npm for the packages!

File: PKGBUILD (hunk 1 above)
Comment (brunnre8) [reply PKGBUILD:6]: I know, guess how many iterations I had to do? 😉 

We anyhow need to adapt it to remove the prerelease string and add the .SRCINFO prior to publishing it

**Comment by brunnre8** [comment]
Thanks for the review.
With this I think we're all set from the arch side, simple version adjustment and srcinfo gen and we're good.

**Pull request merged**